Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Use the official ROS 2 humble base image
+FROM ros:humble
+
+# Set the working directory
+WORKDIR /ros2_workspace/src
+
+COPY . .
+# Install additional dependencies
+RUN apt-get update \
+    && apt-get -y --quiet --no-install-recommends install \
+    gcc \
```

Comment:
ROSdep will install all dependencies needed as shown in the github workflows.

Reply:
As a developer, I use git and other tools at container.And also, rviz2 cant installed by rosdep

Reply:
`ros-humble-ament-cmake-clang-format` is automatically installed by rosdep; adding here is duplicating logic, and ties this dockerfile to `humble` in yet one more spot. Please consider using rosdep for all the dependencies that you can, and then just putting the special dev tools that are missing in the list at the end of the dock

Reply:
Also, gcc is already included by ros-dev-tools.
https://www.ros.org/reps/rep-2001.html#dev-tools

Reply:
You are right I delete installation without rviz2